### DISCUSSION TITLE
i want change rustdesk title message

### DISCUSSION THREAD

**Discussion by SamGenTLEManKaka** [rustdesk/rustdesk#3857] [General]
![524d6b1ed426ea38a5ddecf3d5a0c38](https://user-images.githubusercontent.com/24671601/229136511-6a2e0c94-e768-40b3-939b-64fe1c40414a.jpg)

this is windows agent

i want change this title and icon .
where are these code?
i can not find them

**Comment by JelleBuning** [comment]
Maybe this helps(Changing the application name/icons):
https://github.com/CW3MH6/rustdesk-customization

**Reply by tahssinaldrajee** [reply]
This is what I am looking for, thanks to the questioner and thanks to the respondent.
